spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -64,6 +64,6 @@ public Resource resolve(String location, ResourceLoader resourceLoader) {
 		if (location.startsWith("gs://")) {
 			return new GoogleStorageResource(this.storage, location);
 		}
-		return this.delegate.getResource(location);
+		return null;
```

Comment:
Thank you, @viniciusccarvalho , for the explanation, but to be honest and fully nasty, that really should be reflected in the commit message. Otherwise we will lost ourselves in a mess of commits very quickly and with all these jumping everywhere it's pretty easy to forget "Why?".
Having some explanation in the commit leads me do not pay attention to this change, because I already no "What?" and "Why?"

Just IMO
Thanks